eng/CredScanSuppression.json

```diff
@@ -44,7 +44,8 @@
         "liilef#$DdRGSa_ewkjh",
         "kt#_gahr!@aGERDXA",
         "goose",
-        "_password"
+        "_password",
+        "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjM2MDB9.adM-ddBZZlQ1WlN3pdPBOF5G4Wh9iZpxNP_fSvpF4cWs"
```

Comment:
I assume this is an expired or invalid JWT token correct? Is there a reason it needs to be a JWT token or can we use another fake value from the list here?

Reply:
The decoded json doesn't contain nothing more than the following
```
{"header": {"alg":"HS256","typ":"JWT"}, "payload": {"exp":3600}}
```
It's easier for customer to understand in the context of this example in the README.md

https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/communication/communication-common/README.md#create-a-credential-with-a-static-token